fineftp-server/src/ftp_session.cpp

```diff
@@ -1302,7 +1304,31 @@ namespace fineftp
                                                         }
                                                         else
                                                         {
-                                                          me->sendFtpMessage(FtpReplyCode::CLOSING_DATA_CONNECTION, "Done");
+                                                          // Close Data Socket properly
+                                                          {
+                                                            asio::error_code ec;
```

Comment:
Can you fix the "ec hides previous declaration" warning here?

Reply:
Sure.

Reply:
Actually. Where is that warning coming from? That is, why did you see it? I didn't see that in my builds.

Reply:
The warning came from Visual Studio 2022. It wasn't a problem in this case, but hiding variables can also be unintentional, so I like to keep the project as warning-free as possible.